notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>It would be good to explain in the introduction what QLP is, and what high-level science products are and why the vast majority of the rest of the table may not be useful to the new/learning user.</p><p> </p><p>Are these two QLP time series data products the same as the two data files returned by exo.MAST? Is that the connection?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>